apps/arena/docs/src/attributes/traps.md

```diff
@@ -0,0 +1,45 @@
+# Trap Attributes
```

Comment:
Can you explain shortly what a trap is, and what's their function?

Reply:
Sure!